lib/kryptonite/aes.ex

```diff
@@ -244,4 +346,9 @@ defmodule Kryptonite.AES do
 
   @spec cut_key(binary) :: binary
   defp cut_key(<<key::binary-size(@key_byte_size), _::binary>>), do: key
+
+  defp do_stream_encrypt(elem, acc) do
+    {acc, cypher} = :crypto.stream_encrypt(acc, elem |> List.wrap())
```

Comment:
I wish we could pipe stuff on 2nd argument... Wouldn't an infix library be awesome? Something that would allow:

```
{acc, cypher} =  elem
  |> List.wrap()
  2> :crypto.stream_encrypt(acc)
```